ql/src/test/org/apache/hadoop/hive/ql/TestTxnAddPartition.java

```diff
@@ -102,14 +102,14 @@ private void addPartition(boolean isVectorized) throws Exception {
     String testQuery = isVectorized ? "select ROW__ID, p, a, b from T order by p, ROW__ID" :
         "select ROW__ID, p, a, b, INPUT__FILE__NAME from T order by p, ROW__ID";
     String[][] expected = new String[][]{
-        {"{\"writeid\":1,\"bucketid\":536870912,\"rowid\":0}\t0\t0\t2",
-            "warehouse/t/p=0/delta_0000001_0000001_0000/000000_0"},
-        {"{\"writeid\":1,\"bucketid\":536870912,\"rowid\":1}\t0\t0\t4",
-            "warehouse/t/p=0/delta_0000001_0000001_0000/000000_0"},
-        {"{\"writeid\":1,\"bucketid\":536870912,\"rowid\":0}\t1\t0\t2",
-            "warehouse/t/p=1/delta_0000001_0000001_0000/000000_0"},
-        {"{\"writeid\":1,\"bucketid\":536870912,\"rowid\":1}\t1\t0\t4",
-            "warehouse/t/p=1/delta_0000001_0000001_0000/000000_0"}};
+        {"{\"writeid\":2,\"bucketid\":536870912,\"rowid\":0}\t0\t0\t2",
```

Comment:
I think this is an issue here: Why is the first delta is with writeId = 2?

Reply:
It depends how many DDLs have been executed on a given table.

Reply:
What are the operations happened on the table?

Reply:
Just for my understanding does compactor handle such cases correctly? eg. delta_0000002_0000002_0000, delta_0000004_0000004_0000 is compacted to delta_0000002_0000004_v123?